login-workflow/docs/components/workflow-card.md

```diff
@@ -0,0 +1,74 @@
+| WorkflowCardInstructionProps | `WorkflowCardInstructionProps` | See [WorkflowCardInstructions](#workflowcardinstructions) for more details. |  |
+| WorkflowCardActionsProps | `WorkflowCardActionsProps` | See [WorkflowCardActions](#workflowcardactions) for more details. |  |
+
+### WorkflowCardBaseProps
+
+| Prop Name | Type | Description | Default |
+|---|---|---|---|
+| loading | `boolean` | A boolean that indicates whether the loading spinner should be displayed. | `false` |
+| backgroundImage | `string` | The background image to display behind the card. |  |
+
+The properties of the underlying MUI [Box](https://mui.com/components/box/) component are also available.
+
+### WorkflowCardHeader
+
+See MUI [CardHeaderProps](https://mui.com/components/cards/#cardheader) for more details.
+
+### WorkflowCardBody
+
+This component is a simple wrapper that is used for layout. Instructions and your main screen contents should be children of this component.
+
+### WorkflowCardInstructions
+| Prop Name | Type | Description | Default |
+|---|---|---|---|
+| instructions | `string \| JSX.Element` | The instructions to display in the card. |  |
+| divider | `boolean` | If true, a dividing line will be displayed below the instruction text | `true` |
+
+The properties of the underlying MUI [Typography](https://mui.com/components/typography/) component are also available.
+
+### WorkflowCardActions
+
+The WorkflowCardActions is used to configure the buttons that appear at the bottom of the card.
+
+| Prop Name | Type | Description | Default |
+|---|---|---|---|
+| divider | `boolean` | If true, a dividing line will be displayed above the button panel | `true` |
+| showPrevious | `boolean` | Indicates whether the previous button should be displayed | `true` |
+| canGoPrevious | `boolean \| (() => boolean)` | A boolean or function that indicates whether the previous button should be enabled. | `true` |
+| previousLabel | `string` | The label on the previous button |  |
+| onPrevious |  `() => void` | A function that is called when the previous button is clicked. |  |
+| showNext | `boolean` | Indicates whether the next button should be displayed | `true` |
+| canGoNext | `boolean \| (() => boolean)` | A boolean or function that indicates whether the next button should be enabled. | `true` |
+| nextLabel | `string` | The label on the next button |  |
+| onNext |  `() => void` | A function that is called when the next button is clicked. |  |
+| currentStep | `number` | Indicates the current step in a multi-step workflow | `0` |
+| totalSteps | `number` | Indicates the total number of steps in the multi-step workflow | `0` |
+| fullWidthButton | `boolean` | If true, buttons will expand to the full width of the panel (useful if you only have one button configured) | `false` |
+
+The properties of the underlying MUI [CardActions](https://mui.com/components/card-actions/) component are also available.
```

Comment:
this link is dead for me. should this be https://mui.com/material-ui/api/card-actions/ or https://mui.com/material-ui/react-card/